### ISSUE TITLE
Crown tree representation of Jack Pine

### ISSUE THREAD

**Issue by DylanBernard** [firemodels/fds#13561]
Hello, 

I come to you today because I have a question about the representation of tree in FDS when I use the particle model. 
For more explanation, I use the ‘GENERIC VEGETATION’ describe on the User’s Guide (FDS v6.9.1) for thermal properties of fuel. And the same thing for ‘MOISTURE’, ‘CHAR’ and ‘ASH’. 
In my SURF line description, I use some quantities to describe a Jack Pine (Pinus banksiana Lamb.) and for the PART line description, I use the same quantities than the User’s Guide for DRAG_COEFFICIENT… 

My question is more about the INIT line. Indeed, I have an interrogation about the MASS_PER_VOLUME definition. 
The characteristics of my trees are all described, so I know everything about tree height, crown base height, crown base width and I use the geometry ‘CONE’ to describe my crown. 
In the same article where I have theses information, I have the bulk density of the understory canopy and overstory canopy (I have the profile of bulk density on the height). In my case, I use the data of overstory canopy to represent these crowns. 

Now, when I write this bulk density in the MASS_PER_VOLUME line which represents the mass of fuel for a volume, I have no fire propagation because these data are around 0.2-0.3 kg/m^3. 
So, I’m wondering if these two data are the same or if the bulk density describe in my article aren’t the same that the definition of MASS_PER_VOLUME. 

I’ve tried a lot of parameter changes like MOISTURE_FRACTION, SURFACE_VOLUME_RATIO, … but it’s only when I increase MASS_PER_VOLUME that I get a blaze of fire. 
Concerning my domain, I have a big scale of 100x50x40 m with grid size of 25 cm, a grid size validates in the literature to represent this kind of issue.   
And my plot of experiment has lot of representation of tree, so an important biomass. 

Thank you for your help!

**Comment by ericvmueller** [comment]
Do you have any surface fuel to maintain the fire spread and support a crown fire?

**Comment by mcgratta** [comment]
`MASS_PER_VOLUME` is a measure of the bulk density, that is, the total mass of vegetation contained within a unit volume. If you have not already done this, I suggest that you create a very simple verification case where you set up a single `MESH` at the same resolution as your larger case, say 25 cm. Create a cloud of particles spanning a few cubic meters above a small fire that impinges on the cloud. On the `PART` line, add `QUANTITIES` like `PARTICLE TEMPERATURE`, `PARTICLE DIAMETER`, etc. See what happens. What you want to see is a gradual spreading of the heat through the cloud.

**Comment by mcgratta** [comment]
One thing to add---pay attention to your thermal properties, surface to volume ratio, etc. Moisture. All these things can affect the outcome.

**Comment by DylanBernard** [comment]
> `MASS_PER_VOLUME` is a measure of the bulk density, that is, the total mass of vegetation contained within a unit volume. If you have not already done this, I suggest that you create a very simple verification case where you set up a single `MESH` at the same resolution as your larger case, say 25 cm. Create a cloud of particles spanning a few cubic meters above a small fire that impinges on the cloud. On the `PART` line, add `QUANTITIES` like `PARTICLE TEMPERATURE`, `PARTICLE DIAMETER`, etc. See what happens. What you want to see is a gradual spreading of the heat through the cloud.

I will try this check, thanks for the tips

**Comment by DylanBernard** [comment]
> One thing to add---pay attention to your thermal properties, surface to volume ratio, etc. Moisture. All these things can affect the outcome.

Yes, this is a part of my attention yet. 
Perhaps a problem with my thermal properties, surface to volume ratio and moisture that are from litterature.

**Comment by ericvmueller** [comment]
> Yes normally, there is lot of surface fuel to maintain any fire.

And the issue is that the fire does not spread at all, or that it does not consume the overstory jack pine? The reason I ask is that independent propagation in these low density fuels may be more difficult to predict, and sensitive to the parameters mentioned, but I would hope at least to see fire spread in the surface fuel if you have a reasonable density.

**Comment by DylanBernard** [comment]
> > Yes normally, there is lot of surface fuel to maintain any fire.
> 
> And the issue is that the fire does not spread at all, or that it does not consume the overstory jack pine? The reason I ask is that independent propagation in these low density fuels may be more difficult to predict, and sensitive to the parameters mentioned, but I would hope at least to see fire spread in the surface fuel if you have a reasonable density.

With a reasonable density of surface fuel, indeed I have a propagation near the soil but not a sufficient thermal power to ignite the upper canopy (jack pine) and with difficulty the understory canopy constiting of Black Spruce.

**Comment by ericvmueller** [comment]
Are these data from the ICFME experiments, or similar? I think the range of values you have are representative of the 'canopy bulk density' rather than 'crown bulk density' ...as in, mass per volume over the entire forest canopy at each height. The bulk density in the crown should be higher, since they only occupy a portion of the whole canopy volume. 

If you check out section 3.14 of the FDS Validation guide you can see this distinction is mentioned.

**Comment by DylanBernard** [comment]
> Are these data from the ICFME experiments, or similar? I think the range of values you have are representative of the 'canopy bulk density' rather than 'crown bulk density' ...as in, mass per volume over the entire forest canopy at each height. The bulk density in the crown should be higher, since they only occupy a portion of the whole canopy volume.
> 
> If you check out section 3.14 of the FDS Validation guide you can see this distinction is mentioned.

Exactly, I try to simulate a plot of the ICFME experiments because I study the impact of wildfire on some facilities and these experience are rich of data and results. 
Oh okay, I confused the bulk density of the canopy with the bulk density of the crown. I thought it was very low data for an crown bulk density 

I had already read this section but it seems I didn't understand.

Thanks you for your clarification, it's clear now !
